cmd/firearweave/cli/node.go

```diff
@@ -212,9 +216,20 @@ func (b *bootstrapper) Bootstrap() error {
 
 type nodeArgsByRole map[string]string
 
-func buildNodeArguments(nodeDataDir, nodeRole string, args string) ([]string, error) {
+func buildNodeArguments(dataDir, nodeDataDir, nodeRole string, start, stop uint64, args string) ([]string, error) {
```

Comment:
share `mindreader-node-start-block-num ` and `mindreader-node-stop-block-num` with thegarii